lib/Github/AuthMethod.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace Github;
+
+final class AuthMethod
+{
+    /**
+     * Authenticate using a client_id/client_secret combination.
+     *
+     * @var string
+     */
+    const CLIENT_ID = 'client_id_header';
```

Comment:
Add the `public` const visibility to the constants in this class.

```suggestion
    public const CLIENT_ID = 'client_id_header';
```